packages/govern-console/src/containers/CreateDao/CreateDaoBasicInfo.tsx

```diff
@@ -61,204 +60,184 @@ const CreateDaoBasicInfo: React.FC<{
   };
 
   return (
-    <Grid layout={true}>
-      <GridItem gridColumn={'1/13'} gridRow={'1/4'}>
-        <Box>
-          <div style={{ display: 'grid', gridGap: spacing }}>
-            <Grid columns={'4'} columnWidth={'1fr'}>
-              <GridItem gridColumn={'2/5'}>
-                <Steps steps={stepsNames} activeIdx={0} showProgress={true} />
-              </GridItem>
-              <GridItem gridColumn={'1/2'} gridRow={'1'} alignVertical={'center'}>
-                <StyledText name={'title2'}>Create DAO</StyledText>
-              </GridItem>
-            </Grid>
+    <Box>
+      <div style={{ display: 'grid', gridGap: spacing }}>
+        <Grid columns={'4'} columnWidth={'1fr'}>
+          <GridItem gridColumn={'2/5'}>
+            <Steps steps={stepsNames} activeIdx={0} showProgress={true} />
+          </GridItem>
+          <GridItem gridColumn={'1/2'} gridRow={'1'} alignVertical={'center'}>
+            <StyledText name={'title2'}>Create DAO</StyledText>
+          </GridItem>
```

Comment:
to save you some typing if a property value is a string, i.e., name='title2'

Reply:
noted